frontend/src/ui/Blocks/Series.tsx

```diff
@@ -234,27 +234,40 @@ const VideoGrid: React.FC<React.PropsWithChildren> = ({ children }) => (
 const UpcomingEventsGrid: React.FC<React.PropsWithChildren> = ({ children }) => {
     const { t } = useTranslation();
 
-    return <details css={{
-        backgroundColor: "var(--grey86)",
-        borderRadius: 4,
-        "> summary": {
-            color: "var(--grey20)",
-            cursor: "pointer",
-            fontSize: 14,
-            padding: "6px 12px",
-        },
-        "> hr": {
-            margin: "0 12px",
-        },
-    }}>
-        <summary>
-            {t("series.upcoming-live-streams", { count: Children.count(children) })}
-        </summary>
-        <hr />
-        <VideoGrid>
-            {children}
-        </VideoGrid>
-    </details>;
+    return (
+        <details css={{
+            backgroundColor: "var(--grey86)",
+            borderRadius: 4,
+            marginBottom: "8px",
+            "> summary": {
+                color: "var(--grey20)",
+                cursor: "pointer",
+                fontSize: 14,
+                padding: "6px 12px",
+                "> span": {
+                    marginLeft: "2px",
```

Comment:
Personally I'd prefer 4 or 6 :see_no_evil: 
Also just write `4` instead of `"4px"`.